### ISSUE TITLE
apexrefresh on correct element

### ISSUE THREAD

**Issue by tompetrus** [tompetrus/oracle-apex-custom-chart#2]
The "apexrefresh" event is now bound to the div which is used to contain the chart element, and exists of the region id + "_cc" (cc=CustomChart).
The event should be bound to the actual region. This will enable the region to be refreshed by a dynamic action

**Comment by thomasvanroy** [comment]
is it as simple as removing the custom concatenation of "_cc"?

**Comment by tompetrus** [comment]
No, because I purposefully want to generate the chart element within a div element. There is no control over the chosen region template, and thus it could be an element we don't want. With enclosing in a div we should be safe from this. 
The problem however is that the target id passed in is referring to the "_cc" element. An option is to provide the actual region id as an extra parameter, or creating the div element in the javascript instead. Second option would save the work of doing both things at both sides.
